packages/loader/container-loader/src/retriableDocumentStorageService.ts

```diff
@@ -0,0 +1,58 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { CreateContainerError } from "@fluidframework/container-utils";
+import { IDocumentStorageService } from "@fluidframework/driver-definitions";
+import { canRetryOnError, DocumentStorageServiceProxy } from "@fluidframework/driver-utils";
+import { ISnapshotTree, IVersion } from "@fluidframework/protocol-definitions";
+import { Container } from "./container";
+import { emitThrottlingWarning, getRetryDelayFromError } from "./deltaManager";
+
+export class RetriableDocumentStorageService extends DocumentStorageServiceProxy {
+    private shouldRetry = true;
+    constructor(
+        internalStorageService: IDocumentStorageService,
+        private readonly container: Container,
+    ) {
+        super(internalStorageService);
+    }
+
+    public stopRetry() {
+        this.shouldRetry = false;
```

Comment:
I think it should be named disposed, no?

Reply:
yes, changed api name too.